modules/tensor_mechanics/src/userobjects/CrystalPlasticityStateVarRateComponentGSS.C

```diff
@@ -34,7 +34,7 @@ CrystalPlasticityStateVarRateComponentGSS::calcStateVariableEvolutionRateCompone
 
   Real r = _hprops[0];
   Real h0 = _hprops[1];
-  Real tau_init = _hprops[2];
+//  Real tau_init = _hprops[2];
```

Comment:
That is a good point. I think I began to use this variable for an old formulation and now I do not need this variable. You can leave it as it is now and I will change this as well as some affected tests files by a new PR. Thanks!